src/typescript/azure-openapi-validator/index.ts

```diff
@@ -7,18 +7,23 @@ import { Message } from "../jsonrpc/types";
 import { rules, OpenApiTypes, MergeStates } from "./rule";
 
 // register rules
-require("./rules/DescriptionMustNotBeNodeName");
-require("./rules/ControlCharactersAreNotAllowed");
-require("./rules/ArraySchemaMustHaveItems");
-require("./rules/PostOperationIdContainsUrlVerb");
-require("./rules/LicenseHeaderMustNotBeSpecified");
+let fs = require('fs');
```

Comment:
direct use of `require` in TypeScript is discouraged.  
```  typescript
let fs = require('fs');
let path = require('path');
```

should be at *least* referenced as follows:
``` typescript
import * as fs from 'fs'
import * as path from 'path'
```

And more preferably, just pulling in the functions that are used:
``` typescript
import { join } from 'path'
```